frontend/src/metabase/dashboard/containers/DashboardHeader.jsx

```diff
@@ -209,10 +210,18 @@ class DashboardHeader extends Component {
       toggleSidebar,
       isShowingDashboardInfoSidebar,
       closeSidebar,
+      databases,
     } = this.props;
 
     const canEdit = dashboard.can_write && isEditable && !!dashboard;
 
+    // check the dashboard card's source database,
+    // and check if the database has actions enabled
+    const hasModelActionsEnabled = dashboard.ordered_cards.every(cardData => {
```

Comment:
Adding new actions shouldn't be tight to existing cards on the dashboard. It should be enough to check where there is a database with actions among all databases.

Reply:
Ah ok. From what I understood, the cards (and their source databases) would dictate whether or not there would be an actions button. So instead we will just have the actions button solely based on the database properties?